src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void Test1(){
+        String test = "BYE";
+        assertEquals("bye", Parser.parseCommand(test));
+    }
```

Comment:
I feel that the test description can be more descriptive! Similarly for the rest of the tests.

See https://se-education.org/guides/conventions/java/intermediate.html#naming